Automattic/jetpack
# Blocks: Update fixtures readme and add a check for valid .json in fixture testing

#### Changes proposed in this Pull Request:

* Adds further explanation to the Jetpack blocks fixture tests readme along with an an additional check to automatically check for valid block parsing rather than having to manually check the fixture files.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Check out this PR
- In a block with fixture tests, eg. `slideshow` modify one of the `.html` fixture files to have invalid attributes or markup.
- Run `yarn fixtures:regenerate extensions/blocks/slideshow/test/validate.js` and check that error is thrown about an invalid fixture
- Fix the broken markup and run again and make sure no error is thrown
